pages.pl/osx/say.md

```diff
@@ -0,0 +1,28 @@
+# say
+
+> Czyta na głos.
+> Więcej informacji: <https://ss64.com/osx/say.html>.
+
+- Powiedz na głos:
+
+`say "{{Lubię jeździć na rowerze.}}"`
+
+- Przeczytaj z pliku:
+
+`say --input-file={{plik.txt}}`
```

Comment:
The following code fix could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/say.md) is updated:

```suggestion
`say --input-file={{ścieżka/do/nazwapliku.txt}}`
```

Wrong syntax for path placeholder is used.

Reply:
Fixed